src/entities/PairBurnEvent.ts

```diff
@@ -5,6 +5,34 @@ import { Entity, Column, PrimaryColumn } from 'typeorm';
 export class PairBurnEvent {
   @PrimaryColumn('integer', { generated: true })
   id: number;
```

Comment:
```suggestion
  id?: number;
```
do you think this is necessary?

Reply:
I'm using the entity class for type checks, and I had to make id as an optional parameter because we are not passing the id when saving a new entity.

Reply:
So do you want to make the id's for the rest of the entities optional as well? I noticed you only did it for a few of them

Reply:
Yes, I should. I only did it for the entitles I've used.